Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Multipart build dockerfile to build and serve datagateway
+
+FROM node:16-alpine as build
+WORKDIR /datagateway
+ENV PATH /datagateway/node_modules/.bin:$PATH
+
+# TODO: use yarn install --production:
```

Comment:
Where did we get to with the `use yarn install --production` issue?

Reply:
Was it dependency issues related to the latest version of yarn?

Reply:
I organised the dev and prod dependencies as part of https://github.com/ral-facilities/datagateway/pull/1171 which is needed to successfully do a production install. Once this PR is merged, the install should be succeeding. However, that PR will merge the changes into the `feature/react-17-upgrade-new-#830` branch (PR for it [here](https://github.com/ral-facilities/datagateway/pull/1129)) which deals with a few upgrades, one of which is yarn 2, so we cannot really address the TODO until both of them are merged. I will update the table in the wiki that you created to track this too.